modelling/src/main/java/org/axonframework/modelling/saga/repository/jdbc/GenericSagaSqlSchema.java

```diff
@@ -58,18 +58,21 @@ public GenericSagaSqlSchema(SagaSchema sagaSchema) {
 
     @Override
     public PreparedStatement sql_loadSaga(Connection connection, String sagaId) throws SQLException {
-        final String sql = "SELECT serializedSaga, sagaType, revision FROM " + sagaSchema.sagaEntryTable() + " WHERE sagaId = ?";
+        final String sql = "SELECT " +
+                String.join(", ", sagaSchema.serializedSagaColumn(), sagaSchema.sagaTypeColumn(), "revision")
```

Comment:
Curious: why did you skip the `revision` column, @Blackdread?

Reply:
I had not added it because it already had correct naming.
But let's make it consistent, so it should also be included in the change, I already have made change in my PR -> I did not push yet